scripts/link.js

```diff
@@ -8,9 +8,17 @@ const shell = require('shelljs')
 
 const packagesDir = path.join(__dirname, '..', 'packages')
 const nodeModulesDir = path.join(__dirname, '..', 'packages', 'node_modules')
+const nodeModulesWDIODir = path.join(__dirname, '..', 'packages', 'node_modules', '@wdio')
 
 const packages = shell.ls(packagesDir)
 shell.mkdir(nodeModulesDir)
+shell.mkdir(nodeModulesWDIODir)
 packages.forEach(
-    (pkg) => shell.ln('-s', path.join(packagesDir, pkg), path.join(nodeModulesDir, pkg))
+    (pkg) => shell.ln(
+        '-s',
+        path.join(packagesDir, pkg),
+        pkg.startsWith('wdio-')
+            ? path.join(nodeModulesWDIODir, pkg.replace('wdio-', ''))
```

Comment:
Is there a reason to leave the `wdio-` prefix on the directories or would this be a good time to clean them up by removing that prefix? It's certainly handy in this case, for figuring out where things live, but it seems extra noise in the file naming.

Reply:
No particular reason. I see the prefix in Babel and Jest too. I think it helps to differentiate which packages will be under the `@wdio` NPM org and which are packages without org.